Formula/x/xz.rb

```diff
@@ -2,11 +2,11 @@ class Xz < Formula
   desc "General-purpose data compression with high compression ratio"
   homepage "https://xz.tukaani.org/xz-utils/"
   # The archive.org mirror below needs to be manually created at `archive.org`.
-  url "https://github.com/tukaani-project/xz/releases/download/v5.6.0/xz-5.6.0.tar.gz"
-  mirror "https://downloads.sourceforge.net/project/lzmautils/xz-5.6.0.tar.gz"
-  mirror "https://archive.org/download/xz-5.6.0/xz-5.6.0.tar.gz"
-  mirror "http://archive.org/download/xz-5.6.0/xz-5.6.0.tar.gz"
-  sha256 "0f5c81f14171b74fcc9777d302304d964e63ffc2d7b634ef023a7249d9b5d875"
+  url "https://github.com/tukaani-project/xz/releases/download/v5.6.1/xz-5.6.1.tar.gz"
+  mirror "https://downloads.sourceforge.net/project/lzmautils/xz-5.6.1.tar.gz"
+  mirror "https://archive.org/download/xz-5.6.1.tar.gz/xz-5.6.1.tar.gz"
+  mirror "http://archive.org/download/xz-5.6.1.tar.gz/xz-5.6.1.tar.gz"
```

Comment:
first time uploading it, got it a bit off

<img width="704" alt="image" src="https://github.com/Homebrew/homebrew-core/assets/1580956/8727c1a1-2564-4797-9b29-cd9164fffbaf">